locales/translation-de.json

```diff
@@ -285,15 +288,18 @@
       "name": "Die Adresse, unter der die Gruppe \"@softwerkskammer.org\" erreichbar ist.",
       "title": "Anzeigetext innerhalb der Site, ist auch Realname für E-Mails",
       "x_coord": "Waagerechte Position auf der Karte",
-      "y_coord": "Senkrechte Position auf der Karte"
+      "y_coord": "Senkrechte Position auf der Karte",
+      "contactTheOrganizers": "Auf der Gruppenseite wir ein Kontaktlink angezeigt, über den Besucher ohne Anmeldung Kontakt mit den Ansprechpartnern aufnehmen können."
     },
     "type": "Gruppenart",
     "x_coord": "X-Koord.",
     "y_coord": "Y-Koord."
   },
   "mailsender": {
+    "contact_persons_cannot_be_contacted": "Ansprechpartner können für diese Gruppe nicht kontaktiert werden.",
```

Comment:
Das ist so nicht richtig, denn man kann sie ja immer individuell kontaktieren. Dafür ist ja das "Ansprechpartner"-Label eigentlich gedacht.

Reply:
Was wir im Moment haben ist ein erster Zwischenschritt. Das Ziel ist schon das nicht angemeldete Besucher (Gäste) die Ansprechpartner kontaktieren können. Dafür fehlt aber noch etwas (siehe description)

Reply:
Ich habe mich hier ausschließlich auf das Wording bezogen. "können nicht kontaktiert werden" stimmt nicht. Weder jetzt noch in Zukunft. (Gäste können sich immer anmelden und dann die Ansprechpartner kontaktieren.) Also bitte etwas umformulieren ;-)